### DISCUSSION TITLE
Ability to reverse the sort order of albums

### DISCUSSION THREAD

**Discussion by chipoman** [ente-io/ente#3889] [Enhancements]
All my albums are named by year (though they have not necessarily been created chronologically - for example, I scanned many 1950-70's photos in 2012-14). If I choose sort by name, the 1950's albums come up first and I have to scroll from there to get to my current album! A simple switch to reverse the order would fix it.

**Comment by christkl** [comment]
I fully support this request. When I want to add photos to an album, I would like to see the last used albums on top of the list. Using the search function is a workaround, but it would be great to have the possibility of choosing the sorting order in this dialogue.

**Comment by ua741** [comment]
Dup: https://github.com/ente-io/ente/discussions/1462
